cmd/juju/common/model.go

```diff
@@ -130,6 +138,19 @@ func ModelInfoFromParams(info params.ModelInfo, now time.Time) (ModelInfo, error
 		modelInfo.SLA = ModelSLAFromParams(info.SLA)
 		modelInfo.SLAOwner = ModelSLAOwnerFromParams(info.SLA)
 	}
+
+	if info.CloudCredentialTag != "" {
+		credTag, err := names.ParseCloudCredentialTag(info.CloudCredentialTag)
+		if err != nil {
```

Comment:
I wonder if these things should be fatal errors, or if they just become warnings.
Not a big deal either way.

Reply:
This follows all other tag parsing logic, including tin this method above. I'd leave as is.